aep/AEP-3-dag-generator.md

```diff
@@ -0,0 +1,75 @@
+# AEP-2 DAG Generator
+
+Authors:
+
+* Felix Uellendall (felix.uellendall@astronomer.io)
+
+## Status
+
+| <!-- -->       |  <!-- -->                                                 |
+| -------------- | --------------------------------------------------------- |
+| **State**      | Accepted                                                  |
```

Comment:
Since we already accepted/decided on one of the options I set the state to "Accepted". Please let me know if you think we should have another vote on this.